pkg/gofer/config/origin.go

```diff
@@ -143,6 +145,7 @@ func DefaultOriginSet(pool query.WorkerPool) *origins.Set {
 		"poloniex":      &origins.Poloniex{Pool: pool},
 		"sushiswap":     &origins.Sushiswap{Pool: pool},
 		"uniswap":       &origins.Uniswap{Pool: pool},
```

Comment:
```suggestion
		"uniswap2":       &origins.UniswapV2{Pool: pool},
```

Reply:
Yep I also thought about it but left it because in setzer we do have exact same naming. Didn't want to add differences between them.